### ISSUE TITLE
Data set 'SegmentationClassAug' problem

### ISSUE THREAD

**Issue by yeyuanzheng177** [DrSleep/tensorflow-deeplab-resnet#189]
Thank you for providing the precious code. Your train.txt uses the 'SegmentationClassAug' dataset, but the label for this dataset is different from the semantic split label for pascal voc 2012. For example, the human label in the 'SegmentationClassAug' dataset is (15, 15, 15), which is different from pascal voc 2012. How did you get the 'SegmentationClassAug' data set?

**Comment by DrSleep** [comment]
please refer to the papers mentioned in README
